CMake/cdat_modules/vtk_external.cmake

```diff
@@ -26,21 +26,21 @@ list(APPEND vtk_build_args
 )
 
 # Use cdat zlib
-if(NOT CDAT_USE_SYSTEM_ZLIB)
-  list(APPEND vtk_build_args
-    -DZLIB_INCLUDE_DIR:PATH=${cdat_EXTERNALS}/include
-    -DZLIB_LIBRARY:FILEPATH=${cdat_EXTERNALS}/lib/libz${_LINK_LIBRARY_SUFFIX}
-  )
-endif()
+#if(NOT CDAT_USE_SYSTEM_ZLIB)
```

Comment:
Aashish will comment more, but is there a compelling reason to make these changes in master?